index.html

```diff
@@ -1,11 +1,460 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700;800&family=Roboto:ital,wght@0,500;1,400&family=Rubik:wght@400;500;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>LaslesVPN</title>
 </head>
+
 <body>
-    
+    <header class="header">
+        <div class="wrapper header__wrapper">
+            <a href="/" class="header__home-link">
+                <img src="assets/Logo_vpn.png" alt="logo LaslesVPN">
+            </a>
+            <nav class="header__navigation">
+                <ul class="header__list">
+                    <li class="header__item">
+                        About
+                    </li>
+                    <li>
+                        Features
+                    </li>
+                    <li>
+                        Pricing
+                    </li>
+                    <li>
+                        Testimonials
+                    </li>
+                    <li>
+                        Help
+                    </li>
+                </ul>
+            </nav>
+            <div class="header__signin">
+                <p class="sign-in">
+                    Sign In
+                </p>
+                <button class="nav_button">
+                    Sign Up
+                </button>
+            </div>
+        </div>
+    </header>
+    <main>
+        <!-- hero section -->
+        <section class="hero">
+            <div class="wrapper__hero">
+                <div>
+                    <h1 class="title">
+                        Want anything to be easy with <b>LaslesVPN.</b>
+                    </h1>
+                    <p class="title_description">
+                        Provide a network for all your needs with ease and fun using <b>LaslesVPN</b> discover
+                        interesting features from us.
+                    </p>
+                    <p>
+                        <button class="hero_button">
+                            Get Started
+                        </button>
+                    </p>
+                </div>
+                <img class="hero_img-position" src="assets/hero_img (1).png" alt="title_img">
+            </div>
+            <ul class="hero_info">
+                <li class="hero_info-text">
+                    <button class="hero_info-btn">
```

Comment:
я далі не буду писати "якщо по бему :)"

```suggestion
                    <button class="hero__info-btn">
```